### PR TITLE
[UX] Navigation settings for all devices

### PR DESCRIPTION
**Pull request by Frenzie**
Before multiswipes and the gesture manager this was impractical on touch-only devices, but no more!

### CODE REVIEW

File: frontend/ui/elements/common_settings_menu_table.lua [hunk 1]
```diff
@@ -165,92 +165,91 @@ if Device:isTouchDevice() then
...
+                G_reader_settings:saveSetting("back_to_exit", "disable")
+            end,
+        },
+    },
+}
+common_settings.back_in_filemanager = {
+    text = _("Back in file browser"),
+    sub_item_table = {
+        {
+            text = _("Defer to back to exit setting"),
```
Comment (Frenzie) [inline frontend/ui/elements/common_settings_menu_table.lua]: @poire-z Btw, I'm not too happy about this one but I thought "default" was rather unclear.

Comment (poire-z) [reply frontend/ui/elements/common_settings_menu_table.lua]: Mhhh, indeed, I don't understand it :) Why the "setting" word?
I don't remember how it works, but why not just "Exit" ?

Comment (Frenzie) [reply frontend/ui/elements/common_settings_menu_table.lua]: Because it only exits if the back to exit setting is "exit". ;-)

Comment (Frenzie) [reply frontend/ui/elements/common_settings_menu_table.lua]: It could also be prompt (the default) or ignore.

Comment (poire-z) [reply frontend/ui/elements/common_settings_menu_table.lua]: Oh, ok. I would have understood it better with a capitalized Back as the other item. But you don't like that :) "As in Back to exit", "Same as Back to exit".
No real idea :|

Comment (Frenzie) [reply frontend/ui/elements/common_settings_menu_table.lua]: How about something like this?

## Navigation menu


## Back in file browser submenu

Comment (poire-z) [reply frontend/ui/elements/common_settings_menu_table.lua]: Clever :)

**Pull request merged**